src/carter-mc-barter-narter-is-dope.css

```diff
@@ -1,7 +1,32 @@
-body {
-  min-height: 100vh;
+div{
+  min-height: 5vh;
   display: flex;
   flex-direction: row;
   align-items: center;
   justify-content: center;
 }
+.button{
+  font: roboto;
+  font-size: 18px;
+  cursor: pointer;
+}
+.button:hover{
+  font-size: 24px;
+  text-align: center;
+  background-image: linear-gradient(to left, violet, indigo, blue, green, yellow, orange, red);
+  -webkit-background-clip: text;
+  color: transparent;
+  position: relative;
+  animation: myfirst 5s linear 2s infinite alternate;
+}
```

Comment:
This is so cool! Let's think about naming these styles to be a bit more correctly named - wdyt?

A thought for structuring this entire file:
```
// Common

.flex-centered {
  display: flex;
  flex-direction: row;
  align-items: center;
  justify-content: center;
}

// Page- & component-specific

.main-page-title {
  min-height: 5vh;
  font: roboto;
  font-size: 18px;
}

.main-page-title:hover {
  font-size: 24px;
  text-align: center;
  background-image: linear-gradient(to left, violet, indigo, blue, green, yellow, orange, red);
  -webkit-background-clip: text;
  color: transparent;
  position: relative;
  animation: rainbow-slide 5s linear 2s infinite alternate;
}

@keyframes rainbow-slide {
  0%   {left:0px; top:5px;}
  50% {left:200px; top:5px}
  100% {left:-200px; top:5px}
}
```

Also: don't worry about not getting naming right the first time on these. Naming is, after all, one of the two hardest problems in computer science!

1. Cache invalidation
2. Naming things
3 Off-by-one errors

😉